gcp-cs-network-component/Program.cs

```diff
@@ -0,0 +1,56 @@
+﻿using System.Collections.Generic;
+using System.Threading.Tasks;
+
+using Pulumi;
+using Pulumi.Gcp.Compute;
+using Pulumi.Gcp.Compute.Inputs;
+
+class Program
+{
+    static Task<int> Main()
+    {
+        return Deployment.RunAsync(() =>
+        {
+            string instanceName = "test";
+            var network = new Pulumi.Gcp.Compute.Network(instanceName);
+
+            var firewall = new Pulumi.Gcp.Compute.Firewall(instanceName, new Pulumi.Gcp.Compute.FirewallArgs()
+            {
+                Allows = new InputList<FirewallAllowsArgs>(){
+                    new FirewallAllowsArgs(){
+                        Ports={"22"},
```

Comment:
```suggestion
                        Ports = { "22" },
```

Please adjust spacing everywhere.